components/kyma-environment-broker/cmd/broker/broker_suite_test.go

```diff
@@ -381,7 +388,7 @@ func (s *BrokerSuiteTest) AssertProvisionerStartedProvisioning(operationID strin
 	assert.NoError(s.t, err)
 
 	var status gqlschema.OperationStatus
-	err = wait.Poll(pollingInterval, 2*time.Second, func() (bool, error) {
+	err = wait.Poll(pollingInterval, 2000*time.Second, func() (bool, error) {
```

Comment:
nit: similar here, I'd personally write this as `33 * time.Minute` if we need such long polling

Reply:
fixed